README.rst

```diff
@@ -208,6 +208,34 @@ Non-exhaustive list of Dragonfly modules that should work (with the above change
 - firefox (except save_now command)
 - audacity
 
+Using SSL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Configuring SSL:
+TODO
+
+Creating certificates:
+
+First create a certificate authority::
+
+        $ aenea_cert root my_root
+
+This created the files ``my_root.key`` and ``my_root.pem``.
+
+Now create a server certificate::
+
+        $ aenea_cert --root my_root --ip 192.168.1.78 server my_server
```

Comment:
In that case we need another configuration option (only on the client side I believe). Because it's not just something that matters in certificate creation, it should be set as an option when creating the SSL context. Verifying host names is also possible, I haven't done it before  but I don't think it's very difficult. I could add it as an option.

Maybe the config option should be something along the lines of SSL_VERIFY_MODE being one of none, ip or hostname.